py/picca/fitter2/xi.py

```diff
@@ -142,21 +142,18 @@ def wrapper(*args, **kwargs):
 def growth_factor_de(*args, **kwargs):
     return cached_growth_factor_de(*args, **kwargs)
 
-def cached_growth_factor_de(z, zref = None, **kwargs):
+def cached_growth_factor_de(z, zref = None, Om = None, OL = None, **kwargs):
```

Comment:
Same here. There shouldn't be spaces around the `=` in function parameters

Reply:
done